README.md

```diff
@@ -208,6 +208,13 @@ network:
     version: 2
 ```
 
+For fedora users, you can run the following commands to edit the sysconfig(s) found at `/etc/NetworkManager/system-connections` via nmcli.
+
+Example sysconfig nmcli commands:
```

Comment:
I'm not sure if we should have instructions how to set Pi-hole as the DNS for the host at all. While this is a valid configurations, the host's DNS will break if Pi-hole is not running.

Reply:
I see where you'e coming from. This is a more general discussion. (I think I meant to suggest to also move the Ubuntu part to the docs). Maybe @pi-hole/docker-maintainers have an opinion